docs/index.asciidoc

```diff
@@ -378,6 +379,24 @@ See <<plugins-{type}s-{plugin}-retry_policy,Retry Policy>> for more information.
 
 Timeout (in seconds) to wait for data on the socket. Default is `10s`
 
+[id="plugins-{type}s-{plugin}-ssl_supported_protocols"]
+===== `ssl_supported_protocols`
+
+* Value type is <<string,string>>
+* Allowed values are: `TLSv1.1`, `TLSv1.2`, 'TLSv1.3'
+* There is no default value for this setting
+
+List of allowed SSL/TLS versions to use when establishing a connection to the HTTP endpoint.
+The default depends on the JDK being used and with up-to-date Logstash is `['TLSv1.2', 'TLSv1.3']`.
+`TLSv1.1` is not considered secure and is only provided for legacy applications.
```

Comment:
```suggestion
* Value type is <<string,string>>
* Allowed values are: `TLSv1.1`, `TLSv1.2`, `TLSv1.3`
* Default depends on the JDK being used. With up-to-date Logstash, the default is `['TLSv1.2', 'TLSv1.3']`.
`TLSv1.1` is not considered secure and is only provided for legacy applications.

List of allowed SSL/TLS versions to use when establishing a connection to the HTTP endpoint.
```

Reply:
nope that wasn't - it's a typo, thanks for noticing